### DISCUSSION TITLE
Novas regras de tributação

### DISCUSSION THREAD

**Discussion by lima-emanuel** [DyegoCosta/trabalhando-remoto#197] [General]
Foram aprovadas as novas regras:
https://economia.uol.com.br/noticias/redacao/2021/09/01/camara-reforma-ir-imposto-renda-pessoas-empresas.htm

Pelo que eu entendi, no caso da maior parte de nós, continuaremos isentos de tributação de dividendos e nosso CSLL diminuiu. O que vocês acharam? Entenderam o mesmo que eu também?

**Comment by pablo-sg-pacheco** [comment]
Também entendi a mesma coisa. Provavelmente pra vocês que devem ser Simples Nacional como eu, vamos acabar pagando menos o DAS (Documento de Arrecadação do Simples), pq parece que o CSLL está embutido no DAS

**Comment by penkz** [comment]
Pelo que entendi foi isso também.

`Pelo texto, empresas do Simples Nacional não terão de pagar o tributo (Imposto sobre dividendo). Pequenas empresas, com faturamento de até R$ 4,8 milhões, também ficam isentas, segundo o relatório.`

Só no caso da CSLL que eu não tenho certeza se vai afetar alguma coisa para empresas do Simples, já que o cálculo é no Simples feito de forma diferente, esse foi um dos motivos de não ter havido imposto sobre dividendos...

**Comment by copercini** [comment]
Achei Stonks
`Pelo texto, empresas do Simples Nacional não terão de pagar o tributo` (sobre distribuição de dividendos)
o Imposto de renda do Pró-labore baixa
CSLL acho que vai baixar muito pouco, pois representa uma porcentagem muito pequena no DAS

**Comment by valtermesquitaosf** [comment]
Muita coisa pode mudar, ou nada, quando o texto passar pelo senado. Não da pra se preocupar ainda.
